vscode-lean4/src/infoview.ts

```diff
@@ -27,10 +27,10 @@ class RpcSession implements Disposable {
 
     constructor(client: LeanClient, public sessionId: string, public uri: ls.DocumentUri) {
         this.client = client;
-        this.keepAliveInterval = setInterval(() => {
+        this.keepAliveInterval = setInterval(async () => {
```

Comment:
what is this fixing?

Reply:
There are errors that were marked in the build after upgrading the language client to 8 version. I'm going to revert this and put it in the PR of the language client update